mobile/modules/ui-components/organisms/NetworkSelectionModal3.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useCallback, useRef, useMemo } from 'react'
+import { StyleSheet, View, Text, Button } from 'react-native'
+import { GestureHandlerRootView } from 'react-native-gesture-handler'
+import BottomSheet, { BottomSheetFlatList } from '@gorhom/bottom-sheet'
+
+const NetworkSelectionModal3 = () => {
+  // hooks
+  const sheetRef = useRef<BottomSheet>(null)
+
+  // variables
+  const data = useMemo(
+    () =>
+      Array(50)
+        .fill(0)
+        .map((_, index) => `index-${index}`),
+    []
+  )
+  const snapPoints = useMemo(() => ['25%', '50%'], [])
+
+  // callbacks
+  const handleSheetChange = useCallback((index) => {
+    console.log('handleSheetChange', index)
+  }, [])
+  const handleSnapPress = useCallback((index) => {
```

Comment:
```suggestion
  const handleSnapPress = useCallback((index: number) => {
```
Fix error: Parameter 'index' implicitly has an 'any' type.